boom! testing fern_macros-0.1.7 against beta for beta
boom! running: cargo +beta build --frozen
boom! creating container for: cargo +beta build --frozen
boom! running `docker create -v /home/ec2-user/cargobomb/./work/local/test-source/beta/beta:/source:ro -v /home/ec2-user/cargobomb/./work/local/cargo-home:/cargo-home:ro -v /home/ec2-user/cargobomb/./work/local/rustup-home:/rustup-home:ro -v /home/ec2-user/cargobomb/./work/local/target-dirs/beta/beta:/target:rw -e USER_ID=500 -e CMD=cargo +beta build --frozen cargobomb`
blam! 51ca4b70b1f73272f43be084f1656b77a11443aea310632c56f5fa596f918a5b
boom! running `docker start -a 51ca4b70b1f73272f43be084f1656b77a11443aea310632c56f5fa596f918a5b`
kablam!    Compiling fern v0.1.12
kablam! error[E0432]: unresolved import `std::old_io`
kablam!  --> /cargo-home/registry/src/github.com-1ecc6299db9ec823/fern-0.1.12/src/config.rs:1:5
kablam!   |
kablam! 1 | use std::old_io as io;
kablam!   |     ^^^^^^^^^^^^^^^^^ no `old_io` in the root
kablam! 
kablam! error[E0432]: unresolved import `std::old_io::stdio`
kablam!  --> /cargo-home/registry/src/github.com-1ecc6299db9ec823/fern-0.1.12/src/config.rs:2:5
kablam!   |
kablam! 2 | use std::old_io::stdio;
kablam!   |     ^^^^^^^^^^^^^^^^^^ Could not find `old_io` in `std`
kablam! 
kablam! error[E0432]: unresolved import `std::old_io`
kablam!  --> /cargo-home/registry/src/github.com-1ecc6299db9ec823/fern-0.1.12/src/loggers.rs:1:5
kablam!   |
kablam! 1 | use std::old_io as io;
kablam!   |     ^^^^^^^^^^^^^^^^^ no `old_io` in the root
kablam! 
kablam! error[E0432]: unresolved import `std::old_io::stdio`
kablam!  --> /cargo-home/registry/src/github.com-1ecc6299db9ec823/fern-0.1.12/src/loggers.rs:2:5
kablam!   |
kablam! 2 | use std::old_io::stdio;
kablam!   |     ^^^^^^^^^^^^^^^^^^ Could not find `old_io` in `std`
kablam! 
kablam! error[E0432]: unresolved import `std::old_io`
kablam!  --> /cargo-home/registry/src/github.com-1ecc6299db9ec823/fern-0.1.12/src/errors.rs:1:5
kablam!   |
kablam! 1 | use std::old_io as io;
kablam!   |     ^^^^^^^^^^^^^^^^^ no `old_io` in the root
kablam! 
kablam! error[E0412]: cannot find type `Path` in this scope
kablam!   --> /cargo-home/registry/src/github.com-1ecc6299db9ec823/fern-0.1.12/src/config.rs:35:10
kablam!    |
kablam! 35 |     File(Path),
kablam!    |          ^^^^ not found in this scope
kablam!    |
kablam!    = help: possible candidate is found in another module, you can import it into scope:
kablam!              `use std::path::Path;`
kablam! 
kablam! error[E0412]: cannot find type `Path` in this scope
kablam!   --> /cargo-home/registry/src/github.com-1ecc6299db9ec823/fern-0.1.12/src/loggers.rs:76:29
kablam!    |
kablam! 76 |     pub fn with_file(path: &Path) -> io::IoResult<WriterLogger<io::File>> {
kablam!    |                             ^^^^ not found in this scope
kablam!    |
kablam!    = help: possible candidate is found in another module, you can import it into scope:
kablam!              `use std::path::Path;`
kablam! 
kablam! error[E0405]: cannot find trait `FromError` in module `error`
kablam!   --> /cargo-home/registry/src/github.com-1ecc6299db9ec823/fern-0.1.12/src/errors.rs:11:6
kablam!    |
kablam! 11 | impl error::FromError<io::IoError> for Error {
kablam!    |      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ not found in `error`
kablam! 
kablam! error[E0405]: cannot find trait `FromError` in module `error`
kablam!   --> /cargo-home/registry/src/github.com-1ecc6299db9ec823/fern-0.1.12/src/errors.rs:17:10
kablam!    |
kablam! 17 | impl <T> error::FromError<sync::PoisonError<T>> for Error {
kablam!    |          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ not found in `error`
kablam! 
kablam! error[E0554]: #[feature] may not be used on the beta release channel
kablam!  --> /cargo-home/registry/src/github.com-1ecc6299db9ec823/fern-0.1.12/src/lib.rs:1:1
kablam!   |
kablam! 1 | #![feature(old_io, old_path)]
kablam!   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
kablam! 
kablam! error: aborting due to previous error
kablam! 
kablam! error: Could not compile `fern`.
kablam! 
kablam! To learn more, run the command again with --verbose.
boom! running `docker rm -f 51ca4b70b1f73272f43be084f1656b77a11443aea310632c56f5fa596f918a5b`
blam! 51ca4b70b1f73272f43be084f1656b77a11443aea310632c56f5fa596f918a5b
